src/Cli/dotnet/Commands/Tool/Execute/ToolExecuteCommandParser.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Text;
+using Microsoft.DotNet.Cli.Commands.Tool.Install;
+using NuGet.Packaging.Core;
+
+namespace Microsoft.DotNet.Cli.Commands.Tool.Execute
+{
+    internal static class ToolExecuteCommandParser
+
+    {
+        public static readonly Argument<PackageIdentity?> PackageIdentityArgument = ToolInstallCommandParser.PackageIdentityArgument;
+
+        public static readonly Argument<IEnumerable<string>> CommandArgument = new("commandArguments")
+        {
+            Description = "arguments forwarded to the tool"
+        };
+
+        public static readonly Option<string> VersionOption = ToolInstallCommandParser.VersionOption;
+        public static readonly Option<bool> RollForwardOption = ToolInstallCommandParser.RollForwardOption;
```

Comment:
Why do we have to define these here as opposed to just adding ToolInstallCommandParser.RollForwardOption in ToolExecuteCommandParser.ConstructCommand and using that in the Command?